README.md

```diff
@@ -207,6 +207,45 @@ There is available REST API for AMOC where we can:
 * check status of nodes in AMOC cluster
 
 API is described [here](REST_API_DOCS.md). You can also get an access to REST API by running AMOC and go to `(address:port)/api-docs`.
-### Docker
 
-TODO
+## Docker
+
+Building local Docker image may be started with:
+```
+docker build -f docker/Dockerfile -t tag .
+```
+It is important to start building at project root (it is indicated with dot `.`
```

Comment:
I don't think these two paragraphs are necessary - they describe capabilities of Docker which we're using in the case of Amoc, but aren't specific to it. IMO such information shouldn't be kept in the README.

Reply:
I think we may keep first paragraph, because it describes what are requirements for building Amoc image. The second one could be removed.

Reply:
The only requirement is to be at the project root, which can be described with a single sentence. I just think that the README should be focused on usage, and not internals. Anyway, I'm fine with keeping the first paragraph.